test/resources/payment-draft.json

```diff
@@ -0,0 +1,15 @@
+{
+  "amountPlanned": {
+    "type": "centPrecision",
+    "currencyCode": "EUR",
+    "centAmount": 795,
+    "fractionDigits": 2
+  },
+  "paymentMethodInfo": {
+    "paymentInterface": "ctp-adyen-integration",
```

Comment:
maybe we should use more dummy examples ? 😄

Reply:
It's dummy, but at the same time it is a product placement for our great [adyen-integration project](https://github.com/commercetools/commercetools-adyen-integration)